speech/cloud-client/beta_snippets.py

```diff
@@ -126,6 +127,36 @@ def transcribe_file_with_auto_punctuation(path):
 # [END speech_transcribe_file_with_auto_punctuation]
 
 
+# [START speech_transcribe_word_level_confidence]
+def transcribe_file_with_word_level_confidence(speech_file):
+    """Transcribe the given audio file synchronously with
+      word level confidence."""
+    from google.cloud import speech_v1p1beta1 as speech
+    client = speech.SpeechClient()
+
+    with open(speech_file, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = speech.types.RecognitionAudio(content=content)
+
+    config = speech.types.RecognitionConfig(
+        encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='en-US',
+        enable_word_confidence=True)
```

Comment:
API nit: having to use a flag to enable word confidence when the others are already returned is a bummer.